lib/Cargo.toml

```diff
@@ -61,7 +61,7 @@ bung = "0.1.0"
 channel = { version = "1.8.0", package = "async-channel" }
 chrono = { version = "0.4.24", features = ["serde"] }
 derive = { version = "0.8.0", package = "surrealdb-derive" }
-dmp = "0.1.3"
+dmp = { version = "0.1.3", git = "https://github.com/finnbear/surrealdb_dmp", branch = "bugfix_cleanup" }
```

Comment:
Currently depends on my `dmp` fork. Merging https://github.com/surrealdb/dmp/pull/1 and releasing a new version of `dmp` is recommended.

Reply:
Done @finnbear !